36 time page

AkimZmerli opened the pull request:
## Checklist

Confirm you have completed the following actions prior to submitting this PR.

- [x] There is an existing issue report for this PR.
- [x] I have created a feature branch.
- [x] My changes have been committed.
- [x] I have pushed my changes to the branch.

## Title

Style and Label for TimePage implemented

## Description

The styling from Label Component is used now to show the selected Date and Time 

## Issue Resolution

#36 

Closes #

## Proposed Changes

_List your proposed changes below._

-
-
-

## New or Changed Features

Backend data is needed to implement actual times. Also disabled Time and Dates have not yet been suggested

AkimZmerli committed:
applied style as in Figma

AkimZmerli committed:
added label behaviour

AkimZmerli committed:
resolve merge conflict

dnmct commented on src/routes/TimePage.tsx:
```diff
@@ -0,0 +1,60 @@
+        DATE
+      </div>
+      <DisplayDate today={today} />
+      <div className="border-solid border mt-5 border-white-dimmed"></div>
```
- This is the wrong color
- also your line is 2px thick instead of one

dnmct reviewed (dismissed):
Please change errors and lint warnings, before further review

AkimZmerli committed:
fixed styling

Border thickness reduced from 2 to 1px. changed color to
white-dimmed-heavy. fixed lint errors

AkimZmerli committed:
final commit

louziferr committed:
resolve merge conflict

louziferr committed:
rename label.tsx

louziferr reviewed (approved):
nice job!
